---
content_type: page
description: This section includes lecture topics and summaries.
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Notes
uid: 4976973e-4559-84d1-ce3d-6dad96c01c38
---

All lecture summaries ({{% resource_link 2af264b6-a4de-92a3-d8b1-624418c29e02 "PDF" %}})

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
SES #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction
{{< tdclose >}}
{{< tdopen >}}


Introduction of instructors, students, and course

Overview of translation (ribosome, ribosomal proteins, rRNA, mRNA, tRNA, translation factors, genetic code)

We will watch the amazing mini movie "Elongation Cycle of Protein Biosynthesis" by Agrawal, Whiting & Frank starring the _E. coli_ ribosome, tRNA and elongation factors. Protein synthesis is one of the most fundamental cellular processes in a living cell. Protein synthesis occurs on the ribosome. It is composed of three distinct steps: **Initiation**, **elongation** and **termination**. Although protein synthesis is highly conserved in all kingdoms of life, there are critical structural and functional differences between prokaryotes and eukaryotes.

In this first class, we will review the ordered events in protein synthesis and review the function of the various components of the translation machinery. We will discuss high-resolution structures of the prokaryotic and eukaryotic ribosome obtained by x-ray crystallography and cryo-electron microscopy and how these structures help us understand the mechanism of action of toxins, antibiotics and other protein synthesis inhibitors covered later in the course.

Sit back and enjoy a fantastic voyage through the most incredible molecular machine: The Ribosome!


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Toxins I: Toxins that target the ribosome - Ricin and Shiga toxin
{{< tdclose >}}
{{< tdopen >}}


In 1978, a very small amount of Ricin was used by the KGB to kill a Bulgarian Soviet political dissident in London, and Ricin is still considered to be one of the most serious bio-terrorist threats. Many organisms produce toxic proteins that exert lethal activity towards other cells. Ricin is an extremely potent protein toxin produced by the castor bean plant and has no known antidote. A single molecule will inactivate almost 2000 ribosomes within a minute.

The bacterial Shiga toxin is also extremely potent. Interestingly, both Ricin and Shiga toxin are ribosome-inactivating proteins and target the eukaryotic ribosome large subunit in a similar fashion. We will discuss the experimental techniques and data that explain how Ricin and Shiga toxin specifically target the 60S ribosomal subunit and why they are so potent.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Toxins II: Toxins that target eukaryotic elongation factor 2 - Diphtheria toxin and Pseudomonas exotoxin A
{{< tdclose >}}
{{< tdopen >}}
Diphtheria, a respiratory tract infection, is caused by the gram-positive _Corynebacterium diphtheriae_; the mortality rate of diphtheria is 5-10 %. _Diphtheria_ toxin (DT) is one of the most studied and best-understood bacterial toxins. Both _Diphtheria_ toxin and _Pseudomonas_ exotoxin. A target eukaryotic elongation factor 2 (eEF-2) by covalent modification of diphthamide, a unique modified amino acid in the protein. We will discuss experiments that led to the identification of the cellular targets of these toxins and how resistance to those toxins could arise.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Toxins III: Toxins that target tRNA - Bacterial colicins and the eukaryotic γ- toxin
{{< tdclose >}}
{{< tdopen >}}


Although the majority of protein synthesis inhibitors target the ribosome or protein translation factors, several tRNA-specific toxins have been identified. Action of these toxins often requires the presence of specific base modification on the tRNA. Base modifications are implicated in stability of the 3D-structure of the tRNA molecules, others in the codon-anticodon interactions, or interaction with proteins. Some have yet unknown functions.

In this class, we will discuss the importance of tRNA modifications, and how tRNA-specific toxins including bacterial colicins and the eukaryotic γ-toxin stop protein synthesis in bacteria and yeast, respectively.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Mechanism and action of antibiotics I: Tetracycline and other antibiotics that target the 30S ribosomal subunit
{{< tdclose >}}
{{< tdopen >}}


Tetracycline, a widely used antibiotic, is a good example of drugs that bind directly to the ribosome and thereby interfere with its function. Tetracycline binds to the 30S subunit of the prokaryotic ribosome and blocks binding of the tRNA. The exact mode of action of tetracycline is well understood, especially with the availability of the 3D structure of the ribosome cocrystallized with tetracycline, and the mapping of mutations that confer resistance to tetracycline.

In this class, we will discuss the mode of action of tetracycline and other antibiotics that target the small ribosomal subunit, and highlight differences between these antibiotics.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Mechanism and action of antibiotics II: Chloramphenicol and other antibiotics that target the 50S ribosomal subunit
{{< tdclose >}}
{{< tdopen >}}
Ribosomes are a major target for natural and synthetic antibiotics. Besides being of clinical importance, antibiotics are powerful probes to study ribosome structure and function. Detailed knowledge of the binding sites of these antibiotics on the ribosome (provided by x-ray crystallography, NMR or chemical footprinting) is critical for our understanding of their mechanism at the molecular level. This knowledge will facilitate rational approaches to the design of new antibiotics and the identification of new antibiotic targets on the ribosome of bacterial pathogens to overcome the increasing problem of resistance against currently used antibiotics.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Mechanism and action of antibiotics III: Aminoglycoside antibiotics - Antibiotics that affect translational fidelity
{{< tdclose >}}
{{< tdopen >}}


Many human diseases are caused by nonsense mutations (premature termination codons; PTCs) that lead to the generation of truncated, functionally inactive or less active proteins. One such example is _Duchenne_ muscular dystrophy (DMD). DMD is caused by mutations in the dystrophin gene, leading to the absence of the dystrophin protein in striated muscle cells. Without dystrophin muscle cells gradually die causing a severe weakness of muscle tissue. At present, there is no cure for DMD.

We will discuss the use of aminoglycoside antibiotics (e.g. gentamycin, G-418, paromomycin) as a possible clinical treatment for diseases caused by PTCs. Aminoglycoside antibiotics are known to interact with the highly conserved decoding site of the ribosome where pairing between the codon of the mRNA and the corresponding anticodon of a tRNA (cognate tRNA) is carried out.

When aminoglycoside antibiotics are bound to the ribosome, reduced translational fidelity is observed and near-cognate tRNAs (tRNAs that form two instead of the standard three base-pairs between anticodon and codon) are allowed to participate in translation. It has been shown that, depending on the dose, aminoglycoside antibiotics can be used to partially restore the synthesis of full-length protein from a gene carrying a PTC without inhibiting overall protein synthesis.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Protein engineering I: Incorporation of unnatural amino acids into proteins - Making SENSE out of NON-SENSE
{{< tdclose >}}
{{< tdopen >}}


Incorporation of unnatural amino acids (amino acid analogues) with novel chemical, physical and biological properties at specific sites into proteins adds a new dimension to studies of protein structure, function, protein-protein interactions and protein localization. In addition, it provides a powerful tool for the design of proteins with novel chemical and biological properties.

Unnatural amino acids of interest include those that are photoactivatable or fluorescent; those that carry heavy atoms or spin labels for structural studies such as crystallography, NMR and spectroscopy; those that have reactive side chains such as keto groups; and those that mimic amino acids with phosphorylated or glycosylated side-chains. The demand for _in vitro_ and _in vivo_ synthesis of proteins with modifications at specific sites has driven the development of suppressor tRNA-based methods, as illustrated in this week's papers.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Protein engineering II: Incorporation of unnatural amino acids into proteins - Use of ribosomal frameshift-suppressor tRNAs and editing-defective aminoacyl-tRNA synthetases
{{< tdclose >}}
{{< tdopen >}}


The use of 4- and 5-base frameshift suppressor tRNAs, which decode 4- or 5-base codons in an mRNA and thereby suppress ribosomal frameshifting, provides the means for further expansion of the genetic code. The combination of two frameshift suppressor tRNAs or the combination of nonsense and frameshift suppressor tRNAs allows concomitant incorporation of two different unnatural amino acids.

Many aminoacyl-tRNA synthetases contain a secondary active site for editing mischarged or non-cognate amino acids before they are used in translation. Editing is crucial to maintain amino acid specificity and thereby the accuracy of the genetic code. In this week's second paper, we will discuss how editing-defective aminoacyl-tRNA synthetases can be used to insert amino acid analogues into proteins.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Protein engineering III: In vitro evolution of proteins
{{< tdclose >}}
{{< tdopen >}}


Protein engineering employs two different strategies: (1) Rational design; and (2) Directed evolution where random mutagenesis of a protein gene is followed by an appropriate selection strategy to pick out those variants of the protein that show the desired quality. Over the past years, powerful ribosome display systems were developed that allow _in vitro_ (cell-free) evolution of proteins with novel characteristics. In these systems, the protein and its encoding mRNA remain attached to the ribosome; multiple rounds of selection are performed to enrich for those proteins with new and specific properties, such as protein stability, folding and functional activity.

Ribosome display has a number of advantages over cell-based systems; it can display very large libraries without the restriction of bacterial transformation. It is also suitable for generating toxic or unstable proteins, and furthermore allows the incorporation of modified amino acids at defined positions.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
The translational apparatus and human diseases
{{< tdclose >}}
{{< tdopen >}}


It comes as no surprise that mutations in individual components of the translational machinery are responsible for a wide spectrum of human diseases. Two interesting examples are mutations in tRNAs and aminoacyl-tRNA synthetases (class of enzymes responsible for attaching the correct amino acid onto a tRNA). So far, more than 150 disease-related mutations have been identified within the human mitochondrial genome. Most remarkably, more than half of these mutations are found within tRNA genes and are responsible for diseases such as mitochondrial myopathy, encephalopathy, lactic acidosis, and stroke-like episodes (MELAS). More recently, mutations in aminoacyl-tRNA synthetases have been shown to cause protein misfolding leading to several pathological conditions, including neurodegeneration.

In this class, we will also discuss the possibility of using specifically modified tRNAs as a novel therapeutic approach for diseases caused by premature termination codons in mRNAs of certain genes. This approach could possibly complement treatments that use aminoglycoside antibiotics.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Epilogue
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}